### ISSUE TITLE
Site Audit: Alt text

### ISSUE THREAD

**Issue by ExperimentsInHonesty** [hackforla/website#2370]
### Overview
We need to have our site meet Accessibility standards for alt text so that we can meet our mission of inclusivity.

### Action Items

- [ ] List all site pages for audit below.
- [ ] Review the article for the instructions about which type of images should not have alt text.
- [ ] Review the images on the site and designate which images get alt text and designate this on the Alt text audit tracker.
- [ ] Check off each page in the list below after audit is completed on that page. 
- [ ] Alert dev lead after each page is finished for issue creation. 

### Resources/Instructions
https://www.boia.org/blog/8-common-image-alt-text-mistakes-to-stop-making
#### for future ticket for developers on images that do not need alt text.
```
The image referred to in this issue is a decorative image and requires a null alt attribute (e.g. alt="")
```
[WCAG section on alt text](https://www.w3.org/WAI/WCAG21/Techniques/html/H37.html)
[FAE rule 4: alt text should be less than 100 characters](https://fae.disability.illinois.edu/rulesets/IMAGE_4_EN/)
[Alt text audit tracker](https://docs.google.com/spreadsheets/d/1abMlPOX88zHTV29E4BFxAYwahCxgJN28Mn95cXr8jak/edit#gid=0)
#### List of Pages that need audits:

- [x] Program Areas #2354

**Comment by kristine-eudey** [comment]
Update this issue with all the pages that need audit.

**Comment by IsaacDesigns** [comment]
@ExperimentsInHonesty hse decided that we will divide all pages up between designers and developers in issue https://github.com/hackforla/website/issues/2515.

Then design will annotate the alt text within each page.
